packages/loader/container-loader/src/container.ts

```diff
@@ -1507,13 +1510,15 @@ export class Container
 		}
 
 		this._attachState = AttachState.Attached;
+		const loadStage1 = performance.now() - startTime;
 
 		// Fetch specified snapshot.
 		const { snapshot, versionId } =
 			pendingLocalState === undefined
 				? await this.fetchSnapshotTree(specifiedVersion)
 				: { snapshot: pendingLocalState.baseSnapshot, versionId: undefined };
 
+		startTime = performance.now();
```

Comment:
I'd advise against reporting individual durations for each phase the way you do it as it leaves gaps (for example, we are not recording time between lines 1513 & 1521).
Probably the best course of action is just recording actual values of performance.now() as a set of properties starting with the value at the start of this function. While not the easiest to work with data, we can write Azure quires to extract, do subtraction and report numbers for individual phases.
This way it's also easier if we have optional stages.

I.e. something like that:
```ts
const timings: Record<string, number> = { start: performance.now() };
...
timings.phase2 = performance.now();
...
this.subLogger.sendTelemetryEvent({eventName: "UnaccountedLoadTime", ...timings};
```

I wanted to say - use a single property that is array, but it makes it harder to deal with optional blocks (like block measuring loadStage3). So maybe it's not good idea.